### PR TITLE
translate backend notifications settings

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
translate backend notifications settings and some associated validation responses.

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/Notifications/Apprise/AppriseProxy.cs [hunk 1]
```diff
@@ -19,11 +21,13 @@ public class AppriseProxy : IAppriseProxy
     {
         private readonly IHttpClient _httpClient;
         private readonly Logger _logger;
+        private readonly ILocalizationService _localizationService;
 
-        public AppriseProxy(IHttpClient httpClient, Logger logger)
+        public AppriseProxy(IHttpClient httpClient, Logger logger, ILocalizationService localizationService)
```
Comment (markus101) [inline src/NzbDrone.Core/Notifications/Apprise/AppriseProxy.cs:26]:
```suggestion
        public AppriseProxy(IHttpClient httpClient, ILocalizationService localizationService, Logger logger)
```

File: src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs [hunk 2]
```diff
@@ -28,27 +29,30 @@ public class CustomScript : NotificationBase<CustomScriptSettings>
...
+        private readonly ILocalizationService _localizationService;
 
         public CustomScript(IConfigFileProvider configFileProvider,
             IConfigService configService,
             IDiskProvider diskProvider,
             IProcessProvider processProvider,
             ITagRepository tagRepository,
-            Logger logger)
+            Logger logger,
+            ILocalizationService localizationService)
```
Comment (markus101) [inline src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs:40]:
```suggestion
            ILocalizationService localizationService,
            Logger logger)
```

**Review by markus101 — CHANGES_REQUESTED** [review]
Keeping the `Logger` last keeps it consistent with other files, commented on a couple spots, but applies to all of them.

**Comment by stevietv** [comment]
> Keeping the `Logger` last keeps it consistent with other files, commented on a couple spots, but applies to all of them.

done and rebased

**Comment by markus101** [comment]
Needs a small fix for the code formatting:
`.\src\NzbDrone.Core\Notifications\Join\JoinProxy.cs(27,48): error SA1001: Commas should be followed by whitespace`

**Comment by stevietv** [comment]
> Needs a small fix for the code formatting: `.\src\NzbDrone.Core\Notifications\Join\JoinProxy.cs(27,48): error SA1001: Commas should be followed by whitespace`

fixed! forgot to check back if this one passed the build in all the v4 excitement :D

**Review by markus101 — APPROVED** [review]

**Pull request merged**